config/options.go

```diff
@@ -624,28 +607,6 @@ func (o *Options) Validate() error {
 	return nil
```

Comment:
validation needs to be updated to check the `autocert` option instead of just `len(o.Certificates) == 0`